services/credentials-service/src/credentials/credentials.controller.ts

```diff
@@ -116,4 +116,17 @@ export class CredentialsController {
     return this.credentialsService.verifyCredential(credId);
   }
 
+  @Get('revocation-list')
```

Comment:
Is this part of the specification. Response Type is missing.

Reply:
The endpoint is as per the specification. And I have added the Response Type.